src/Traits/StringTypeTrait.php

```diff
@@ -37,32 +37,35 @@ trait StringTypeTrait
      * Check whether this expression the given expression.
      *
      * @param StringType $right
+     * @param bool $insertParentheses
      * @return Contains
      */
-    public function contains(StringType $right): Contains
+    public function contains(StringType $right, bool $insertParentheses = true): Contains
```

Comment:
The `$insertParentheses` parameter should also be added to the relevant interface (`StringType`).

Reply:
fixed in 03246a1